### PR TITLE
Fixing some code-typos and adding an additional example in README

### PR DESCRIPTION
**Pull request by RockKeeper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,39 @@ requestBody := gsclient.IPCreateRequest{
 client.CreateIP(ctx, requestBody)
 ```
 
+For updating/scaling server resources you could use:
+
+```go
+myServerUuid := "[Server UUID]"
+backgroundContext := context.Background()
+
+// No hotplug available for scaling resources down, shutdown server first via AHCI
```
Comment (fkr) [inline README.md]: Shouldn't this read "ACPI" not "AHCI" (AHCI is Advanced Host Controller Interface and typically refers to the SATA PCI bridge...doesn't it?)

File: README.md [hunk 2]
```diff
@@ -75,6 +75,39 @@ requestBody := gsclient.IPCreateRequest{
...
+}
+
+updateErr := client.UpdateServer(backgroundContext, myServerUuid, requestBody)
+if updateErr != nil{
+	log.Error("Serverupdate failed", updateErr)
+	return
+}
+
+// Start server again
+poweronErr := client.StartServer(backgroundContext, "70ba2037-2acb-407e-88e5-e752f9fa3b0e")
```
Comment (fkr) [inline README.md]: probably meant to use 'myServerUuid' instead of the hardcoded uuid?

File: README.md (hunk 1 above)
Comment (RockKeeper) [reply README.md]: :man_facepalming: it's ACPI for sure

File: README.md (hunk 2 above)
Comment (RockKeeper) [reply README.md]: :man_facepalming: :man_facepalming: yes, of course

**Comment by RockKeeper** [comment]
Both comments changed

**Comment by fkr** [comment]
Awesome. Thanks for the PR!

**Pull request merged**